File: cpp/do_while.cpp
#include <iostream>
using namespace std;

int main() {
  int nilai_awal = 1;
  int nilai_akhir;

  cout << "Nama\t: Yogi Prasetyawan Hadi" << endl;
  cout << "NIM\t: 191110236" << endl;
  cout << "-----------------------------------" << endl;

  cout << "Masukkan Nilai Akhir : ";
  cin >> nilai_akhir;

  do {
    cout << "Nilai ke " << nilai_awal << endl;
    nilai_awal++;
  } while(nilai_awal <= nilai_akhir);

  return 0;
}

File: 3_OOP/P3/src/com/company/DoWhile.java
package com.company;

public class DoWhile {
    public static void main(String args[]) {
        showName();
        int n = 10;
        do{
            System.out.println("Nilai n:"+n);
            n--;
        } while(n >= 1);
    }

    private static void showName() {
        System.out.println("Nama : Yogi Prasetyawan Hadi");
        System.out.println("NIM : 191110236");
        System.out.println("----------------------");
    }
}
